Makefile

```diff
@@ -431,7 +435,7 @@ merge-to-master-release:
 push-to-manifest-repo:
 	@rm -rf $(MANIFESTS_TMP) || true
 	@mkdir -p ${MANIFESTS_TMP}/${BUNDLE_VERSION}
-	operator-sdk generate csv --csv-version $(BUNDLE_VERSION) --from-version=$(BASE_BUNDLE_VERSION)
+	operator-sdk generate csv --csv-version $(BUNDLE_VERSION) --from-version=0.0.23
```

Comment:
Is there a reason to not use `$(BASE_BUNDLE_VERSION)` ?

Reply:
It seems that this has been changed in c3b49bafe199760a0ef446015a627572c6a662d8 already in master.

Reply:
Adding to Igor, BASE_BUNDLE_VERSION ?= $(OPERATOR_VERSION) and we want the generated manifests desc, and other metadata from the version in olm/catalog. Hence the change.